### ISSUE TITLE
[Feature request] give more options for adjustable colors

### ISSUE THREAD

**Issue by makedir** [JasonStein/Notepads#1069]
Following issue especially on dark mode:

- Selected line highlight color is hard to read with dark mode:

![image](https://user-images.githubusercontent.com/6721083/159598004-2b338bb3-3bde-485b-aa76-25862c6194d5.png)

- selected text is hard to read for some accent colors: 

![image](https://user-images.githubusercontent.com/6721083/159598072-72e7743c-2143-4058-8c03-c8a24528ad60.png)

Suggestion:

Give adjustable colors for:

- selected line background color
- flip color for selected text => make the selected text dark when selected

**Comment by JasonStein** [comment]
Currently the text selection color is equal to the theme color which you can change in the settings. It is possible to do just in time color calculation, but I am just too lazy to do so. But for now, I would suggest you pick a different theme color for the app in the settings.

**Comment by makedir** [comment]
That is not a solution. There could be at least 3 or 4 adjustable colors in the settings:

- selected line bg color
- selected text bg color
- selected text font color
- option to flip font color when selected (chose color when flip)

**Comment by JasonStein** [comment]
I think flipping the color of the selected text is a good option, but it should really be flipping the color and choose a close one by gray scale. Meaning there should be some calculation going to find a matching opposite color in gray scale.

**Comment by makedir** [comment]
Yes, of course. But the flip should always be between a dark and non dark color. Example:

![image](https://user-images.githubusercontent.com/6721083/162551587-c65a0c9e-7e9d-4bd3-a101-298f58082722.png)

![image](https://user-images.githubusercontent.com/6721083/162551593-08f06fd6-16cb-4363-b8f1-d055ed86c10b.png)

This would be a perfect readable way how to do it. So like I said, give option for the user to decide what is best. Just give more colors to select. Examples above.

**Comment by ownlyme** [comment]
The contrast in dark mode doesn't cut it for me, is there really no way to change the background color?
